BuildAssets/OctopusTools.nuspec

```diff
@@ -15,9 +15,10 @@
     <releaseNotes></releaseNotes>
     <language>en-US</language>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <licenseUrl>https://github.com/OctopusDeploy/OctopusCLI/blob/master/LICENSE.txt</licenseUrl>
+    <license type="expression">Apache-2.0</license>
```

Comment:
`licenseUrl` is deprecated: https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5125